samba/docker-compose.yml

```
@@ -0,0 +1,30 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: samba_homepage_1
+      APP_PORT: 9445
```

I know it is confusing sometimes, but the `APP_PORT` does not need to be the same as the port in the `umbrel-app.yml`. Therefore I suggest changing this to the default value of the `static-web-server` to get rid of one environment variable
```suggestion
      APP_PORT: 80
```

---

Got it! I had this impression when checking other apps, the external port that the user will access is defined only on umbrel-app.yml.